src/typescript/frontend/src/components/pages/emojicoin/components/main-info/MainInfo.tsx

```diff
@@ -40,28 +37,30 @@ const MainInfo = ({ data }: MainInfoProps) => {
     }
   }, [stateEvents]);
 
-  const { ref: marketCapRef } = useLabelScrambler(marketCap);
-  const { ref: dailyVolumeRef } = useLabelScrambler(dailyVolume);
-  const { ref: allTimeVolumeRef } = useLabelScrambler(allTimeVolume);
+  const { ref: marketCapRef } = useLabelScrambler(toCoinDecimalString(marketCap, 2));
+  const { ref: dailyVolumeRef } = useLabelScrambler(toCoinDecimalString(dailyVolume, 2));
+  const { ref: allTimeVolumeRef } = useLabelScrambler(toCoinDecimalString(allTimeVolume, 2));
 
   return (
-    <div className="flex justify-center">
-      <div className={innerWrapper}>
-        <div className={headerWrapper}>
-          <div
-            title={emojisToName(data.emojis).toUpperCase()}
-            className=" text-white uppercase ellipses display-4 font-forma-bold md:display-2"
-          >
-            {emojisToName(data.emojis)}
-          </div>
-
-          <div className="text-[24px] md:display-2 my-auto text-white">
-            {data.symbolData.symbol}
-          </div>
-        </div>
+    <div
+      className="flex justify-center mt-[10px]"
+      style={{
+        borderTop: `1px solid ${theme.colors.darkGray}`,
+      }}
+    >
+      <div
+        style={{
+          display: "grid",
+          gridTemplateColumns: "57fr 43fr",
+          width: "100%",
+          maxWidth: "1362px",
+          padding: "40px",
+        }}
+      >
+        <div className="text-[100px] text-center my-auto text-white">{data.symbolData.symbol}</div>
 
-        <div className={statsWrapper}>
-          <div className="flex gap-[8px]">
+        <div className="flex flex-col mt-[-8px] ml-[4em] w-fit gap-[2px]">
+          <div className="flex justify-between">
             <div className={statsTextClasses + " text-light-gray"}>{t("Mkt. Cap:")}</div>
```

Comment:
```suggestion
            <div className={statsTextClasses + " text-light-gray"}>{t("Market Cap:")}</div>
```

so they're all aligned on the left as well now (since it's justify right now)